docs/example/mattermost-installation.md

```diff
@@ -268,6 +233,73 @@ kubectl delete action mattermost-instance -n $NAMESPACE
 helm delete -n $NAMESPACE $(helm list -f="mattermost-*" -q -n $NAMESPACE)
 ```
 
+
+### Install Mattermost with an external AWS RDS database
+
+To change the Mattermost installation, we need to adjust our Global policy to prefer AWS solutions. Read more about Global policy configuration [here](../feature/policies/global-policy.md).
+
+#### Instructions
+
+1. Create a TypeInstance with AWS Credentials. See how to create Type Instances with [AWS credentials](typeinstances.md#aws). 
+   Make sure that environment variable `TI_ID` exists.
```

Comment:
``````suggestion
1. Create AWS Credentials TypeInstance.

   1. Follow the [AWS Credentials TypeInstance creation](./typeinstances.md#aws-credentials) guide to create and obtain ID of the newly created TypeInstance. 
   1. Export the TypeInstance ID as environment variable:

    ```bash
    export TI_ID={AWS credentials TypeInstance ID}
    ```
``````